app/controllers/donations_controller.rb

```diff
@@ -18,7 +19,7 @@ def create
         UserMailer.coordinator_email(@coordinators, @donation).deliver
         format.html { redirect_to root_path, notice: "Donation was successfully created." }
       else
-        format.html { render donations_new_path }
+        format.html { redirect_to root_path, notice: "Donation failed because of unsupported file type." }
```

Comment:
seems to work!!!

however change it from a notice to an alert, so the notice color is red instead.